iptables/iptables.go

```diff
@@ -248,7 +248,12 @@ func (ipt *IPTables) runWithOutput(args []string, stdout io.Writer) error {
 	}
 
 	if err := cmd.Run(); err != nil {
-		return &Error{*(err.(*exec.ExitError)), stderr.String()}
+		switch err.(type) {
+		default:
+			return err
+		case *exec.ExitError:
+			return &Error{*(err.(*exec.ExitError)), stderr.String()}
```

Comment:
if you put a variable in on line 251, e.g. `switch e := err.(type)` then you don't need to repeat the cast here.